lib/tasks/csv_load.rake

```diff
@@ -90,6 +90,14 @@ namespace :csv_load do
     end
   end
 
+  desc "Scorched Earth" 
+  task :pk_reset => :environment do
+    ActiveRecord::Base.connection.tables.each do |t|
+      ActiveRecord::Base.connection.reset_pk_sequence!(t)
+    end
+  end
+
   desc "csv load all"
-  task :all => [:customers, :invoice_items, :invoices, :items, :merchants, :transactions]
-end
\ No newline at end of file
+  task :all => [:customers, :invoices, :merchants, :items, :transactions, :invoice_items, :pk_reset]
+end
```

Comment:
This is super awesome that this works thank you for figuring this out and working through it.